python/core/auto_generated/mesh/qgsmeshtimesettings.sip.in

```diff
@@ -0,0 +1,88 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/core/mesh/qgsmeshtimesettings.h                                  *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
+class QgsMeshTimeSettings
+{
+%Docstring
+
+Represents a mesh time settings for mesh datasets
+
+.. versionadded:: 3.8
+%End
+
+%TypeHeaderCode
+#include "qgsmeshtimesettings.h"
+%End
+  public:
+    QDomElement writeXml( QDomDocument &doc ) const;
```

Comment:
Should these methods also be taking a QgsReadWriteContext for possible future use?